rust/operator-binary/src/csi_server/node.rs

```diff
@@ -92,6 +98,12 @@ enum PublishError {
         path: PathBuf,
     },
 
+    #[snafu(display("file path {} must not contain more than one component", path.display()))]
+    InvalidComponentCount { path: PathBuf },
+
+    #[snafu(display("file path {} must not be absolute", path.display()))]
+    InvalidAbsolutePath { path: PathBuf },
```

Comment:
Nit: We should probably quote these (yes, I realize the other paths in the errors aren't quoted either :/)

```suggestion
    #[snafu(display("file path {path:?} must not contain more than one component"))]
    InvalidComponentCount { path: PathBuf },

    #[snafu(display("file path {path:?} must not be absolute"))]
    InvalidAbsolutePath { path: PathBuf },
```

Reply:
Yeah I guess we should. I can also quote the existing ones for consistency.

Reply:
Done in a9be60f6237d1ea61eb3495bb68bc16020d1e136.